feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/repository/AccountRepositoryImpl.kt

```diff
@@ -260,6 +262,25 @@ class AccountRepositoryImpl(
         }
     }
 
+    override suspend fun generateRestoreJson(
+        metaAccount: MetaAccount,
+        password: String,
+    ): String {
+        return withContext(Dispatchers.Default) {
+            val secrets = secretStoreV2.getMetaAccountSecrets(metaAccount.id)!!
+
+            jsonSeedEncoder.generate(
+                keypair = secrets.keypair(ethereum = false),
+                seed = secrets.seed,
+                password = password,
+                name = metaAccount.name,
+                multiChainEncryption = metaAccount.multiChainEncryption()!!,
+                genesisHash = "",
```

Comment:
I dont think we can leave the genesis hash empty - can you check whether such JSON is compatible with Import to Nova and Import to Polkadot.js?

Reply:
Thank you for advice!

It's compatable. I also checked that JSON imported from polkadot.js also doesn't have genesisHash